src/Neo/Network/P2P/Payloads/NotaryAssisted.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// NotaryAssisted.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo.IO;
+using Neo.Json;
+using Neo.Persistence;
+using Neo.SmartContract.Native;
+using System.IO;
+using System.Linq;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class NotaryAssisted : TransactionAttribute
+    {
+        /// <summary>
+        /// Native Notary contract hash stub used until native Notary contract is properly implemented.
+        /// </summary>
+        private static readonly UInt160 notaryHash = UInt160.Parse("0xc1e14f19c3e60d0b9244d06dd7ba9b113135ec3b");
```

Comment:
This will be changed when Notary comes, isn't it?

Reply:
Only in that it'll be some `NativeContract.Notary.Hash` like `GAS.Hash`, but the value will be the same.

Reply:
And now this can probably be more of `Helper.GetContractHash(UInt160.Zero, 0, "Notary")` with the same value, but in a bit easier to digest form.

Reply:
> And now this can probably be more of

Fixed, but I've kept the hard-coded hash value in test to ensure hashes are match. For the sake of my personal compatibility paranoia.